modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/accessor/ExperimentTagAccessor.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright 2017 Intuit
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.intuit.wasabi.repository.cassandra.accessor;
+
+import com.datastax.driver.core.Statement;
+import com.datastax.driver.mapping.Result;
+import com.datastax.driver.mapping.annotations.Accessor;
+import com.datastax.driver.mapping.annotations.Query;
+import com.intuit.wasabi.repository.cassandra.pojo.index.ExperimentTagsByApplication;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Accessor for the tags associated with Experiments.
+ */
+@Accessor
+public interface ExperimentTagAccessor {
+
+    @Query("select * from experiment_tag where app_name IN ?")
```

Comment:
@bargenilesh is this also true for such a small table? It only contains one row per application (we currently have 278 and we only store those which have tags associated with them) and usually users will only have access to a handful of those. It feels like having now one call for each application could slow things down, but maybe I am getting something wrong. I am also not sure about the partitioning in general: if I want the complete table (since it is so small) to be replicated across the nodes, how would I do that? Or does that make no sense? Thanks!